pkg/systemd/manifest.json.in

```diff
@@ -27,6 +27,12 @@
         "services": {
             "label": "Services",
             "order": 100,
+            "docs": [
+                {
+                    "name": "Services",
+                    "url": "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/managing-services-in-the-web-console_system-management-using-the-rhel-8-web-console"
```

Comment:
I'm a bit sceptical that this is a stable link. Did you talk to Vendula about that? I think she had a solution for that.

Reply:
I did talk to her and this indeed is a stable link. It is constructed from chapters and sections IDs.